### PR TITLE
etcdctlv3: interactive_v3 version and put command update

### PR DESCRIPTION
**Pull request by sinsharat**
1. added version section
2. added put command lease example.

### CODE REVIEW

File: Documentation/dev-guide/interacting_v3.md [hunk 1]
```diff
@@ -8,6 +8,18 @@ By default, etcdctl talks to the etcd server with the v2 API for backward compat
 export ETCDCTL_API=3
 ```
 
+## Find the ETCD version
```
Comment (xiang90) [inline Documentation/dev-guide/interacting_v3.md]: ## Find versions

File: Documentation/dev-guide/interacting_v3.md [hunk 2]
```diff
@@ -8,6 +8,18 @@ By default, etcdctl talks to the etcd server with the v2 API for backward compat
...
+
+ETCD version and API version can be useful in finding the appropriate commands to be used for performing various opertions on ETCD.
```
Comment (xiang90) [inline Documentation/dev-guide/interacting_v3.md]: `etcdctl` version and `Server API` version

Comment (xiang90) [inline Documentation/dev-guide/interacting_v3.md]: always use etcd not ETCD

File: Documentation/dev-guide/interacting_v3.md [hunk 3]
```diff
@@ -8,6 +8,18 @@ By default, etcdctl talks to the etcd server with the v2 API for backward compat
...
+
+Here is the command to find the version:
```
Comment (xiang90) [inline Documentation/dev-guide/interacting_v3.md]: the versions

File: Documentation/dev-guide/interacting_v3.md (hunk 1 above)
Comment (sinsharat) [inline Documentation/dev-guide/interacting_v3.md]: sure will modify it.

File: Documentation/dev-guide/interacting_v3.md (hunk 2 above)
Comment (sinsharat) [inline Documentation/dev-guide/interacting_v3.md]: ok sure will keep in mind from next time. Will do the changes.

File: Documentation/dev-guide/interacting_v3.md [hunk 4]
```diff
@@ -19,6 +31,17 @@ $ etcdctl put foo bar
 OK
 ```
 
+Also a key can be set for a specified interval of time by attaching lease to it.
+
+Here is the command to set the value of key `foo1` to `bar1` for 10ms.
```
Comment (xiang90) [inline Documentation/dev-guide/interacting_v3.md]: use 10 seconds as a better example. lease is not designed for high resolution.

File: Documentation/dev-guide/interacting_v3.md (hunk 3 above)
Comment (sinsharat) [inline Documentation/dev-guide/interacting_v3.md]: ok will modify it.

File: Documentation/dev-guide/interacting_v3.md (hunk 4 above)
Comment (sinsharat) [inline Documentation/dev-guide/interacting_v3.md]: sure will modify it.

**Comment by sinsharat** [comment]
@xiang90 I have done the changes as per your suggestion.
Thank you!

File: Documentation/dev-guide/interacting_v3.md [hunk 5]
```diff
@@ -8,6 +8,18 @@ By default, etcdctl talks to the etcd server with the v2 API for backward compat
 export ETCDCTL_API=3
 ```
 
+## Find versions
+
+etcdctl version and Server API version can be useful in finding the appropriate commands to be used for performing various opertions on ETCD.
```
Comment (xiang90) [inline Documentation/dev-guide/interacting_v3.md]: change ETCD to etcd?

Comment (sinsharat) [inline Documentation/dev-guide/interacting_v3.md]: yes i had just realized my mistake. Already done :-)

**Comment by xiang90** [comment]
lgtm

**Pull request merged**